cli/src/main/kotlin/OrtMain.kt

```diff
@@ -168,7 +169,9 @@ object OrtMain : CommandWithHelp() {
 
     private fun showVersionHeader(commandName: String?) {
         val env = Environment()
-        val variables = env.variables.entries.map { (key, value) -> "$key = $value" }
+
+        val variables = mutableListOf("$ORT_USER_HOME_ENV = ${getUserOrtDirectory()}")
+        env.variables.entries.mapTo(variables) { (key, value) -> "$key = $value" }
```

Comment:
This replicates the formatting of an entry to string. Consider adding a MapEntry and then format all entries in one go instead.

Reply:
I can't easily do that as the `ORT_USER_HOME` environment variable, in contrast to the other environment variables, is not always set, but I still always want to show the value of `getUserOrtDirectory()`.